### PR TITLE
Add dropdown list of countries to select for the country field

### PR DESCRIPTION
**Pull request by IsankaSR**
Screenshot from 2021-09-22 13-52-05

### Related Issues
- Issue `#1` or (None)

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [x] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- Related PR `#1` or (None)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [x] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/authentication-portal/src/main/webapp/includes/country-dropdown.jsp [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+</div>
+
+<script defer>
+
+    $(document).ready(function() {
+        $("#country-dropdown").dropdown('hide');
+    });
+
+</script>
+
```
Comment (brionmario) [inline apps/authentication-portal/src/main/webapp/includes/country-dropdown.jsp:71]: Lets remove the extra spaces.

File: apps/authentication-portal/src/main/webapp/includes/country-dropdown.jsp [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    <div class="menu">
+        <c:forEach items="<%=getCountryList()%>" var="country">
+            <div class="item" data-value="${country.value}">
+                <i class="${country.key.toLowerCase()} flag"></i>${country.value}
+            </div>
+        </c:forEach>
+    </div>
+</div>
+
+<script defer>
```
Comment (brionmario) [inline apps/authentication-portal/src/main/webapp/includes/country-dropdown.jsp:64]: Do we need the `defer` here.
```suggestion
<script type="application/javascript">
```

File: apps/authentication-portal/src/main/webapp/includes/country-dropdown.jsp (hunk 1 above)
Comment (IsankaSR) [reply apps/authentication-portal/src/main/webapp/includes/country-dropdown.jsp]: Fixed with 165f11c16e7b9f60a59db290953a01f56526889b

File: apps/authentication-portal/src/main/webapp/includes/country-dropdown.jsp (hunk 2 above)
Comment (IsankaSR) [reply apps/authentication-portal/src/main/webapp/includes/country-dropdown.jsp]: Fixed with 165f11c16e7b9f60a59db290953a01f56526889b

**Review by brionmario — APPROVED** [review]

**Pull request merged**